sphinx/ext/autodoc/__init__.py

```diff
@@ -620,6 +621,8 @@ def get_object_members(self, want_all: bool) -> Tuple[bool, ObjectMembers]:
         If *want_all* is True, return all members.  Else, only return those
         members given by *self.options.members* (which may also be none).
         """
+        warnings.warn('The implementation of Documenter.get_object_members() will be '
+                      'removed from Sphinx-6.0.', RemovedInSphinx60Warning)
```

Comment:
The `Documenter.get_object_members()` has been used since its beginning. So it would be better to make the deprecation period long. So I'll keep this while two years (4.x and 5.x).